CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.22.5]
+----------
+* bump enterprise to 4.22.5
```

Comment:
Please add a more descriptive explanation of what is included in the new release